src/NuGet.Core/NuGet.Packaging/Signing/Verification/IPackageSignatureVerifier.cs

```diff
@@ -2,14 +2,24 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace NuGet.Packaging.Signing
 {
+    /// <summary>
+    /// Loads trust providers and verifies package signatures.
+    /// </summary>
     public interface IPackageSignatureVerifier
     {
-        Task<VerifySignaturesResult> VerifySignaturesAsync(ISignedPackageReader package, CancellationToken token, Guid parentId);
+        /// <summary>
+        /// Verifies package signature.
+        /// </summary>
+        /// <param name="package">Package to be verified.</param>
+        /// <param name="settings">SignedPackageVerifierSettings to be used when verifying the package.</param>
+        /// <param name="token">Cancellation Token.</param>
+        /// <param name="parentId">Guid of the parent event.</param>
```

Comment:
`parentId` is too cryptic a name.  The parameter is for telemetry.  Update comment and parameter name.  Also, seems like it should be nullable with a `null` default if you call this method and don't care about telemetry.

    Guid? telemetryOperationId = null

Reply:
Implementation of the interface has a default value for cases [here](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/Signing/Verification/PackageSignatureVerifier.cs#L28). I will change the name.